### PR TITLE
Add file system utils

### PR DESCRIPTION
**Pull request by gnufied**
We need file system utilities for wiping away filesystem.

### CODE REVIEW

**Review by j-griffith — APPROVED** [review]

File: Dockerfile.openshift [hunk 1]
```diff
@@ -6,6 +6,7 @@ RUN CGO_ENABLED=0 GOOS=${OS} GOARCH=${ARCH} go build -a -ldflags '-extldflags "-
 
 FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
 COPY --from=builder /go/src/sigs.k8s.io/sig-storage-local-static-provisioner/_output/linux/amd64/local-volume-provisioner /local-provisioner
+RUN yum install -y e2fsprogs xfsprogs
```
Comment (jsafrane) [inline Dockerfile.openshift]: `&& yum clean all && rm -rf /var/cache/yum`

Comment (gnufied) [reply Dockerfile.openshift]: fixed

Comment (j-griffith) [reply Dockerfile.openshift]: gahhh!

**Pull request merged**